.editorconfig

```diff
@@ -0,0 +1,16 @@
+# top level config
+root = true
+
+# All files:
+# utf-8
+# Unix-style newlines
+# Remove end-of-file newlines
```

Comment:
Yes, my mistake
On Sun, Sep 4, 2016 at 11:53 AM Josh Holland <notifications@github.com>
wrote:

> In .editorconfig
> <https://github.com/HabitRPG/habitrpg/pull/7957#discussion_r77455556>:
>
> > @@ -0,0 +1,16 @@
> > +# top level config
> > +root = true
> > +
> > +# All files:
> > +# utf-8
> > +# Unix-style newlines
> > +# Remove end-of-file newlines
>
> Given line 13, shouldn't this be something like "ensure file ends with a
> newline"?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7957/files/9535aa9f23d6060ac99f82cc63d71c8ba11df736#r77455556>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_85OrkKS-USkapmVQVh5O6lsQHKPCks5qmveQgaJpZM4Jx4Ke>
> .
>